tests/fuzz/LD_PRELOAD/global_overrides.c

```diff
@@ -48,10 +48,11 @@ int s2n_stuffer_send_to_fd(struct s2n_stuffer *stuffer, int wfd, uint32_t len)
     }
 
     /* Otherwise, call the original s2n_stuffer_send_to_fd() */
-    typedef int (*orig_s2n_stuffer_send_to_fd_func_type)(struct s2n_stuffer *stuffer, int wfd, uint32_t len);
+    typedef int (*orig_s2n_stuffer_send_to_fd_func_type)(struct s2n_stuffer *stuffer, int wfd, uint32_t len, ssize_t *bytes_sent);
     orig_s2n_stuffer_send_to_fd_func_type orig_s2n_stuffer_send_to_fd;
     orig_s2n_stuffer_send_to_fd = (orig_s2n_stuffer_send_to_fd_func_type) dlsym(RTLD_NEXT, "s2n_stuffer_send_to_fd");
```

Comment:
This should really be checked for null:

From https://linux.die.net/man/3/dlsym
> If the symbol is not found, in the specified library or any of the libraries that were automatically loaded by dlopen() when that library was loaded, dlsym() returns NULL.

Reply:
Done.